ui/src/components/forms/ContactForm.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react'
+import './forms.scss'
+
+export function ContactForm() {
+  function encode(data) {
+    return Object.keys(data)
+      .map((key) => encodeURIComponent(key) + '=' + encodeURIComponent(data[key]))
+      .join('&')
+  }
+
+  const [state, setState] = useState({})
+  const [error, setError] = useState('')
+  const options = [
+    {
+      label: 'Consultation',
+      value: 'consultation',
+    },
+    {
+      label: 'Digital Menus',
+      value: 'digital menus',
+    },
+    {
+      label: 'Other',
+      value: 'other',
+    },
+  ]
+
+  const handleChange = (e) => {
+    if (error) {
+      setError('')
+    }
+    setState({ ...state, [e.target.name]: e.target.value })
+    console.log(e.target.name, e.target.value)
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    const form = e.target
+    fetch('/', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      body: encode({
+        'form-name': form.getAttribute('name'),
+        ...state,
+      }),
+    })
+      .then((result) => {
+        console.log('result', result)
+        navigate(form.getAttribute('action'))
```

Comment:
This could be fun, instead of navigating to a different page, you could make a div animate in place of the form that says thank you we will get back to you shortly. you would use state and make an isThankYouShowing var and set it if this triggers